image/descriptor.go

```diff
@@ -41,8 +41,8 @@ func listReferences(w walker) (map[string]*v1.Descriptor, error) {
 		}
 
 		for i := 0; i < len(index.Manifests); i++ {
-			if index.Manifests[i].Annotations["org.opencontainers.ref.name"] != "" {
-				refs[index.Manifests[i].Annotations["org.opencontainers.ref.name"]] = &index.Manifests[i]
+			if index.Manifests[i].Annotations["org.opencontainers.image.ref.name"] != "" {
+				refs[index.Manifests[i].Annotations["org.opencontainers.image.ref.name"]] = &index.Manifests[i]
```

Comment:
Since #163 bumped image-spec to 1.0.0, you can use [`AnnotationRefName`][1] instead of a string literal.

[1]: https://github.com/opencontainers/image-tools/blob/42f749be774e8577745a767d3a96f44eee4a5639/vendor/github.com/opencontainers/image-spec/specs-go/v1/annotations.go#L49